tests/vendor/vendor.json

```diff
@@ -2,6 +2,10 @@
 	"comment": "",
 	"ignore": "test appengine",
 	"package": [
+		{
```

Comment:
This looks like the result of an incorrect govendor invocation.  We should remove this.

Reply:
Fixed.